creator-node/default-config.json

```diff
@@ -24,7 +24,7 @@
   "delegateOwnerWallet": "",
   "delegatePrivateKey": "",
 
-  "ethProviderUrl": "http://docker.for.mac.localhost:8546",
+  "ethProviderUrl": "http://audius_ganache_cli_eth_contracts:8545",
```

Comment:
i think these may need to be added to the hosts.js file @hareeshnagaraj. basically all non cnodes are missing

Reply:
this. is unneeded per offline conversation